mrtt-ui/src/views/SiteQuestionsOverview/MonitoringFormsList.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import MONITORING_FORM_CONSTANTS from '../../constants/monitoringFormConstants'
+import { RowSpaceBetween } from '../../styles/containers'
+import { TableAlertnatingRows, WideTh } from '../../styles/table'
+import { Link, SmallUpperCase } from '../../styles/typography'
+
+const MonitoringFormsList = ({ monitoringFormsSortedByDate, siteId }) => {
+  return (
+    <TableAlertnatingRows>
+      <tbody>
+        {monitoringFormsSortedByDate.map(({ id: formId, form_type, monitoring_date }) => {
+          const formLabel = MONITORING_FORM_CONSTANTS[form_type].label
+          const formUrl = `/sites/${siteId}/form/${MONITORING_FORM_CONSTANTS[form_type].urlSegment}/${formId}`
+          return (
+            <tr key={formId}>
```

Comment:
MUI already built in table styling we could have used but not a huge deal

Reply:
Good to know. Will skip it though since this TableAlternatingRows is used elsewhere and I wanna avoid refactors for non functional stuff at this point.